.travis.yml

```diff
@@ -42,6 +42,7 @@ install:
 # the main build
 script:
   - |
+      rustc --print cfg -Ctarget-cpu=native &&
```

Comment:
I get this is only for reporting? There is no need to `cargo build` with `-Ctarget-cpu=native` hereafter, as those are only for compile time optimization?

Reply:
Yes this is just for reporting, so we can see what kind of machine the travis test is running on!

I'm not that interested in `-Ctarget-cpu=native` at the moment, since we now use runtime target feature detection to deliver good performance by default instead. I keep an eye on that performance doesn't *regress* if the user suppiles `-Ctarget-cpu=native`.